### PR TITLE
test(mme): Add mock GTP tunnel for unit test

### PR DESCRIPTION
**Pull request by ssanadhya**
Signed-off-by: Shruti Sanadhya <ssanadhya@fb.com>

## Summary

Add mock GTP tunnel so that SPGW unit tests can proceed without OVS interaction.
Removed the code guards done with compile-time flag MME_UNIT_TEST from sgw_handlers.c and sgw_task.c.

## Test Plan

make test_oai

make integ_test TESTS=s1aptests/test_attach_detach.py
make integ_test TESTS=s1aptests/test_attach_ul_udp_data.py

### CODE REVIEW

**Review by ulaskozat — APPROVED** [review]
LGTM. Do we want the mock implementation under the gtpv1-u directory?

File: lte/gateway/c/core/oai/tasks/gtpv1-u/gtp_tunnel_mock.c [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/**
```
Comment (ulaskozat) [inline lte/gateway/c/core/oai/tasks/gtpv1-u/gtp_tunnel_mock.c]: the mock itself is now counted for coverage if it is under this folder.

Comment (ssanadhya) [reply lte/gateway/c/core/oai/tasks/gtpv1-u/gtp_tunnel_mock.c]: Good catch @ulaskozat ! I will move it to `test/mock_tasks`

Comment (ssanadhya) [reply lte/gateway/c/core/oai/tasks/gtpv1-u/gtp_tunnel_mock.c]: @ulaskozat , please review latest commit https://github.com/magma/magma/pull/10702/commits/32e6039a290b9fff35e4a9dcb31308f13a83fd74 for suggested changes

File: lte/gateway/c/core/oai/tasks/gtpv1-u/CMakeLists.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ set(GTPV1U_SRC
     gtpv1u_task.c
     gtp_tunnel_openflow.c
     gtp_tunnel_upf.c
+    ${PROJECT_SOURCE_DIR}/test/mock_tasks/gtp_tunnel_mock.c
```
Comment (ulaskozat) [inline lte/gateway/c/core/oai/tasks/gtpv1-u/CMakeLists.txt:6]: we should also be able to remove this from here and instead update the cmake files in the test directory.

**Pull request closed without merging**